internal/audit/auditlog.go

```diff
@@ -24,11 +24,19 @@ import (
 	"github.com/sonatype-nexus-community/nancy/buildversion"
 )
 
+func isEntryValid(params ...interface{}) bool {
+	for _, v := range params {
+		if v == nil {
+			return false
+		}
+	}
+	return true
+}
+
```

Comment:
This will work fine for what we are doing, but if you are curious you can take a look at this for some cases where it WONT work: https://mangatmodi.medium.com/go-check-nil-interface-the-right-way-d142776edef1

Also, this sent me on a quick lil check on if golang likes yoda conditions, and it does not! https://staticcheck.io/docs/checks#ST1017

Reply:
Would you like me to update this? I can also move this function out to somewhere where we can use it everywhere instead of just audit